core/src/impl/Kokkos_SharedAlloc.hpp

```diff
@@ -65,7 +65,7 @@ class SharedAllocationHeader {
  private:
   using Record = SharedAllocationRecord<void, void>;
 
-#ifdef KOKKOS_ARCH_VEGA
+#if defined(KOKKOS_ARCH_VEGA) || defined(KOKKOS_ARCH_NAVI)
```

Comment:
Is that something that will need more investigation or that is definitely the right tuned value?

Reply:
We don't know. The larger value is in principle the safer value, but wastes memory if not needed.

Reply:
RDNA 1 architecture has only 128 byte L2 cache line size. But this is RDNA 2 and I couldn't find a similar in-depth architecture whitepaper for it.

Reply:
@arghdos can you advise here?